test/e2e/framework.go

```diff
@@ -113,6 +122,42 @@ func NewFramework(t ginkgo.GinkgoTInterface) *Framework {
 		httpsURLBase = "https://127.0.0.1:9443"
 	}
 
+	var (
+		kubeConfig  string
+		contourHost string
+		contourPort string
+		contourBin  string
+	)
+	if inClusterTestSuite {
```

Comment:
will add to this soon, captured in https://github.com/projectcontour/contour/issues/3772